Header header

#Left Sensor (0 - 1024)
int16 left

#Right Sensor (0 - 1024)
int16 right


